openlibrary/macros/StarRatings.html

```diff
@@ -1,7 +1,7 @@
 $def with (work_key, edition_key=None, rating=None)
 
 <form id="ratingsForm" method="POST" action="$(work_key)/ratings.json"
-      roperty="reviewRating" typeof="Rating" vocab="http://schema.org/" typeof="Books">
+      property="reviewRating" typeof="Rating" vocab="http://schema.org/" typeof="Books">
```

Comment:
@mekarpeles there's two `typeof` s here too, that doesn't look right either

Reply:
Not sure -- it should probably just be Rating.